### PR TITLE
Update README.md to reflect new TSC

### PR DESCRIPTION
**Pull request by tdcmeehan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,13 @@ The current members of the Presto TSC are:
 
 | Name | Github | Term begins | Term ends | Affiliation |
 | ---- | ------ | ------------|-------------|-----------|
-| Bin Fan  | apc99 | November 1, 2021 | November 1, 2022 | Alluxio |
-| James Sun  | highker | November 1, 2021 | November 1, 2022 | Facebook |
+| James Sun  | highker | January 23, 2023 | November 1, 2022 | Meta |
```
Comment (rschlussel) [inline README.md:33]: fix term end to 2023

File: README.md [hunk 2]
```diff
@@ -30,13 +30,13 @@ The current members of the Presto TSC are:
...
+| Deepak Majeti  | majetideepak | January 10, 2023 | November 1, 2023 | Ahana Cloud |
 | Rebecca Schlussel | rschlussel | November 1, 2021 | May 1, 2023 | Facebook |
-| Rongrong Zhong | rongrong | November 1, 2021 | May 1, 2023 | Facebook |
-| Ying Su  | yingsu00 | November 1, 2021 | November 1, 2022 | Ahana Cloud |
-| Zhenxiao Luo  | zhenxiao | November 1, 2021 | May 1, 2023 | Twitter |
-| Tim Meehan  | tdcmeehan | November 1, 2021 | November 1, 2022 | Facebook |
+| Rongrong Zhong | rongrong | November 1, 2021 | May 1, 2023 | Meta |
```
Comment (rschlussel) [inline README.md:36]: should we update Rongrong's affiliation?

File: README.md (hunk 1 above)
Comment (rschlussel) [inline README.md:33]: maybe order alphabetically.  It was alphabetical before except for your name.

File: README.md (hunk 2 above)
Comment (tdcmeehan) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (tdcmeehan) [reply README.md]: Done!

**Review by rschlussel — APPROVED** [review]

**Review by majetideepak — APPROVED** [review]
+1

**Pull request merged**